Equilobe.DailyReport.SL/AdvancedSettingsDataService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Equilobe.DailyReport.DAL;
+using Equilobe.DailyReport.Models.Interfaces;
+using Equilobe.DailyReport.Models.Policy;
+using Equilobe.DailyReport.Models.Web;
+using Equilobe.DailyReport.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Equilobe.DailyReport.SL
+{
+    public class AdvancedSettingsDataService : IAdvancedSettingsDataService
+    {
+        #region IAdvancedSettingsDataService implementation
+        public List<SourceControlOptions> GetAllReposSourceControlOptions(long instanceId)
+        {
+            var advancedReportSettings = GetInstancePolicyStrings(instanceId);
+
+            if (!advancedReportSettings.Any())
```

Comment:
no need to do this check. if the list is empty, then by default the ToList will return an empty list